packages/editor/src/components/provider/index.native.js

```diff
@@ -161,6 +164,24 @@ class NativeEditorProvider extends Component {
 			// purposes and will be replaced with actual logic in a later PR.
 			this.setState( { isHelpVisible: true } );
 		} );
+
+		// Request current block impressions from native app
+		requestBlockTypeImpressions( ( storedImpressions ) => {
+			const impressions = { ...NEW_BLOCK_TYPES, ...storedImpressions };
```

Comment:
I wanted to highlight the following scenarios with regard to updating the list of "new" blocks. I think the current behavior here is perfectly reasonable though.
- after shipping a release to users, if the need arose, we cannot remove the "sparkle" badge from a block in a subsequent version unless we make changes in the parent apps (or de-register the block completely)
- also, after shipping a release, we cannot force the "sparkle" badge to continue being shown to users on a particular block by increasing or resetting the block's impression count

Reply:
Thank you for stating these limitations, @guarani. From my understanding of the project requirements, I do believe these are OK.